protobuf/closure_proto_js_library.bzl

```diff
@@ -0,0 +1,68 @@
+# -*- mode: python; -*-
+#
+# Copyright 2016 The Closure Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities for building JavaScript Protocol Buffers.
+"""
+
+load("//closure/compiler:closure_js_library.bzl", "closure_js_library")
+
+def closure_proto_js_library(
```

Comment:
In this case, absolutely. The `closure_template_*_library` macro is also going to be rewritten as a Skylark rule in the future. The only reason it's a macro, is because I just copied the legacy code outside of g3 in the interest of saving time. But still, optional. If you don't do it, I probably will in the future. But you would benefit from learning more about Skylark if you haven't already. It's pretty cool.